cluster/gce/manifests/etcd.manifest

```diff
@@ -64,13 +64,17 @@
       }
     ],
     "livenessProbe": {
-      "httpGet": {
-        "host": "127.0.0.1",
-        "port": {{ etcd_livenessprobe_port }},
-        "path": "/health"
+      "exec": {
+        "command": [
```

Comment:
it relies on the fact that etcdctl is installed in the container. It's true, but maybe requires a comment?

Reply:
I added comment to Dockerfile